jetty-cdi/pom.xml

```diff
@@ -19,13 +19,13 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty.servlet</groupId>
-      <artifactId>jetty-webapp-common</artifactId>
+      <groupId>org.eclipse.jetty.servlet6</groupId>
+      <artifactId>jetty-webapp6</artifactId>
```

Comment:
looks good for now. but not sure we want this module depending on specific servlet version